client/components/tooltip/style.scss

```diff
@@ -72,16 +74,21 @@
 		}
 
 		&__tooltip {
-			// Specific styles for the click tooltip variant.
+			// Click tooltips have a light background and dark text.
 			position: relative;
 			padding: 12px;
 			color: $gray-900;
 			text-align: left;
 			border-radius: 2px;
 			border: 1px solid $gray-400;
-			background: $white;
+			background-color: $white;
 			box-shadow: 0 2px 6px 0 rgba( 0, 0, 0, 0.05 );
 
+			// Links inside click tooltips should have a dark color against the light background.
+			a {
+				color: var( --wp-admin-theme-color, $gutenberg-blue );
```

Comment:
Using the original colors used before the recent inadvertent change in #9273

Reply:
```suggestion
				color: $gutenberg-blue;
```

I wonder if we should just set the link color statically like so, without involving the `--wp-admin-theme-color` variable into the equation.

[We set the tooltip background to be `$white` statically](https://github.com/Automattic/woocommerce-payments/blob/ed68ce24c15d667e0ed50ab68a3e629b57df3541/client/components/tooltip/style.scss#L84). If the link color is dynamic, depending on the value of that `--wp-admin-theme-color` variable, there is a possibility where the link color is not readable. For example, if somehow that variable is set to `$white` (maybe for dark mode, color theming, or some other reason), we'll have white link on top of a white background.

What do you think?

Reply:
Yep, it looks especially weird when we have another element within the tooltip coloured blue, and the links coloured by the theme colour.

![image](https://github.com/user-attachments/assets/6544e85d-ad00-46a3-bfd3-9d034ae615e0)

Reply:
The entire payments overview page is a mismatch of theme colours and hard-coded colours. I'll open a new issue to rectify this.

<img width="710" alt="image" src="https://github.com/user-attachments/assets/21bb89c8-4778-41cd-b936-4a9de7d8b0f3">

Reply:
I think it should be sufficient to support these default themes – and for colour consistency, we should prefer these theme colours.

For this PR, I say we stick with the theme colour (previous behaviour before bug was introduced) and make a clear decision on which colour to use in the subsequent colour consistency issue.

<img width="2011" alt="image" src="https://github.com/user-attachments/assets/8151d6cc-1453-48c6-a342-a5ea10318393">

Here's how the tooltip links look with each:

![image](https://github.com/user-attachments/assets/ba3544dd-f415-4dc2-b73d-01434f27d5d1)
![image](https://github.com/user-attachments/assets/e0e01ebd-be52-4911-a00d-8f63480dbbac)
![image](https://github.com/user-attachments/assets/2bf4a72e-fb3a-4c38-9e44-d7f818407128)
![image](https://github.com/user-attachments/assets/d0da43c3-02b7-4537-8bc6-e1d93e6fd542)
![image](https://github.com/user-attachments/assets/478f59f6-db37-4618-86b1-a8450d9ebb4e)
![image](https://github.com/user-attachments/assets/c03ad85a-e536-4f3f-9dd7-ff6d22450d4a)
![image](https://github.com/user-attachments/assets/8dac8d67-9e0b-42b6-b95f-0b4866c3418d)
![image](https://github.com/user-attachments/assets/b6a15ce1-7aef-4166-aea4-bbdff37557bc)
![image](https://github.com/user-attachments/assets/b4ce3126-571e-4a77-920e-27f8c5c8927b)
![image](https://github.com/user-attachments/assets/3a7a17b7-d644-4c7c-9621-2c5da9e6b265)

Reply:
But yes, other WP themes extend these admin colour themes, which we'll need to keep in mind.

For example, Tsubaki, which was the default WooExpress theme, defaults to this "Classic Dark" theme but adds other options:

![image](https://github.com/user-attachments/assets/89ba5047-34e4-4615-9394-dc3c0d4582c6)
<img width="1907" alt="image" src="https://github.com/user-attachments/assets/25f00d2f-d112-441e-81f3-5b53b12b3665">
![image](https://github.com/user-attachments/assets/22695c45-9620-489e-add0-0a1c94afc0f5)

Reply:
#9584 created to address the inconsistent colours when a custom admin theme is enabled